addon/controllers/project/errors.js

```diff
@@ -0,0 +1,83 @@
+import Controller from "@ember/controller";
+import { inject as service } from "@ember/service";
+import { tracked } from "@glimmer/tracking";
+import {
+  lastValue,
+  restartableTask,
+  enqueueTask,
+} from "ember-concurrency-decorators";
+
+export default class ProjectErrorsController extends Controller {
+  @service constructionProject;
+  @service building;
+  @service buildingEntrance;
+  @service dwelling;
+  @service intl;
+  @service notification;
+
+  @tracked projectErrorCount = 0;
+  @tracked buildingErrorCount = {};
+  @tracked entranceErrorCount = {};
+  @tracked dwellingErrorCount = {};
+
+  @tracked buildings = {};
+
+  @lastValue("fetchProject") project;
+  @restartableTask
+  *fetchProject() {
+    const project = yield this.constructionProject.getFromCacheOrApi(
+      this.model.projectId
+    );
+
+    this.projectErrorCount = project.errorList?.length ?? 0;
+
+    return project;
+  }
+
+  @enqueueTask
+  *fetchModel(buildingId) {
+    // building response only contains entrances
+    const building = yield this.building.getFromCacheOrApi(buildingId);
+    this.buildingErrorCount[buildingId] = building.errorList?.length ?? 0;
+    // trigger an update in template, tracking does not work with object properties
+    this.buildingErrorCount = this.buildingErrorCount; // eslint-disable-line no-self-assign
```

Comment:
```suggestion
    this.buildingErrorCount[buildingId] = { ...this.buildingErrorCount, [buildingId]: building.errorList?.length ?? 0 };
```
This updates and refreshes the template in one line. You can maybe also use this for the other use cases later down.